hashtheplanet/resources/git_resource.py

```diff
@@ -51,7 +52,8 @@ def get_all_files_from_commit(commit: Commit) -> List[Tuple[FilePath, BlobHash]]
         for blob in commit.tree.traverse():
             if S_ISDIR(blob.mode):
                 continue
-            file_list.append((blob.path, blob.hexsha))
+            if any(ext in blob.path for ext in list_ext):
+                file_list.append((blob.path, blob.hexsha))
```

Comment:
This could add files with names `hello.html.php4`. Is it ok ?

Reply:
Fixed, The list of extensions is replaced by a Regex